content/docs/getting-started/cms-set-up.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Set-up the CMS
+prev: /docs/getting-started/introduction
+next: /docs/getting-started/edit-content
+---
+
+In this step, we will install `tinacms`, create a CMS instance, wrap our app in the `TinaProvider`, and configure the CMS options.
+
+## Install _tinacms_
+
+Install the `tinacms` package. This is the core package that you will need anytime you're using Tina. You'll also need to install `styled-components` to render the editing UI properly.
```

Comment:
> This is the core package that you will need anytime you're using Tina.

I think this is slightly inaccurate, but I'm not sure if it would be worth explaining the nuance in this beginner-level tutorial

Reply:
Suggestions?

Reply:
> This is the main package that you will use to create your CMS